src/commands/QuotesCommand/QuotesCommand.js

```diff
@@ -0,0 +1,86 @@
+const BaseCommand = require('../../utils/structures/BaseCommand');
+const fs = require('fs');
+const { writeToFile } = require('./utils');
+
+const filePath = 'src/commands/QuotesCommand/quotes.json';
+
+module.exports = class QuotesCommand extends BaseCommand {
+    constructor() {
+        super('quote', 'textAnswers', []);
+
+        this.quotes = {};
+        fs.readFile(filePath, 'utf8', (err, contents) => {
+            if (err) {
+                console.error(err);
+                return;
+            }
+            this.quotes = JSON.parse(contents);
+            console.log(this.quotes);
+        });
+    }
+
+    async run(client, message, args) {
+        if (args[0]) {
+            if (this.quotes[args[0].toLowerCase()])
```

Comment:
maybe create a variable to hold the lower case value? And you can also use it below on the switch statement to allow the user to specify "add", "ADD", "aDD", whatever

Reply:
Also, rewrite this if statement to "return" and avoid the "else" branch, as I explained below for line 80